docs/source/integrations/plugins-event-reference.md

```diff
@@ -0,0 +1,321 @@
+---
```

Comment:
Hard to review these files in depth but looks mostly moved and the changes I observed seemed good.

Reply:
Ah yes sorry, I barely edited this to account for the diff-iculties.